main/serializers/base_resource_serializers.py

```diff
@@ -311,6 +328,18 @@ def get_queryset(self):
         ).distinct()
 
 
+class VeryShortBaseCollectionSerializer(serializers.ModelSerializer):
```

Comment:
```suggestion
class VeryShortCollectionSerializer(serializers.ModelSerializer):
```
je ne vois pas pourquoi il y aurait `Base` dans le nom, c'est juste un sérialiseur de collection (certes utilisés dans les bases, mais il pourrait être utilisé ailleurs)

Reply:
en effet je corrige